docs/rewriter/examples/erfgelu.py

```diff
@@ -0,0 +1,93 @@
+"""Onnx Pattern Rewriting.
+
+This script shows how to define a rewriting rule based on patterns.
+
+First a dummy model with a GELU activation
+===================
+"""
+
+
+import math
+import numpy as np
+import onnx
+import onnx.helper as oh
+import onnx.numpy_helper as onh
+
+import onnxscript
+from onnxscript.rewriter import pattern
+
+
+def original_model():
```

Comment:
I realize that this is pre-existing code. But I suggest using onnxscript to create the model.